### PR TITLE
Correct some grammar faults in the introduction

### PR DESCRIPTION
**Pull request by SeraphJACK**

### CODE REVIEW

File: specification/synopsis.markdown [hunk 1]
```diff
@@ -26,25 +26,25 @@ cloud storage services such as Dropbox and MediaFire and use link-shortener
...
+comes from the authentic source is still a burden, especially for an absolute
+novice. Further, Modded Minecraft faces the global audience, even though English is
 the conventional lingua franca in this community, users who are not proficient
 in English would still be annoying.
 
 Inspired by various package manager used by different Linux Distributions as
 well as the CKAN Project from Kerbel Space Program community, the Schema for
 Minecraft Mod Artifact Resource Tracking (SMMART) intends to set up a uniformed
-and platform-agnostic standard to record and exchange information about mod
+and platform-agnostic standard to record and exchange information about the mod
```
Comment (Yesterday17) [inline specification/synopsis.markdown]: is `the` here necessary?

**Review by Yesterday17 — COMMENTED** [review]
(

File: specification/synopsis.markdown (hunk 1 above)
Comment (SeraphJACK) [reply specification/synopsis.markdown]: idk... ask 3t(

Comment (3TUSK) [reply specification/synopsis.markdown]: "mod files" here does not refer to a *specific* mod, so no.

**Pull request merged**